github-flow-sync-with-mirror/index.js

```diff
@@ -12,7 +12,7 @@ const {
 
 async function run() {
   try {
-    const mergeBranch = 'bot-sync-with-mirror';
+    const mergeBranch = 'fix/bot-sync-with-mirror';
```

Comment:
Should this be `chore/` rather than `fix/` ?

Reply:
Thanks fixed